bin/migration/tables/audits.py

```diff
@@ -23,7 +25,10 @@ def migrate_data(self, destination_cursor, source_data):
                 category =audit_log[20]
                 activity = audit_log[2]
                 functional_area = audit_log[17]
-                audit_details = f'\{"description": "{audit_log[5]}, {audit_log[18]}, {audit_log[19]}"\}'
+
+                audit_details_dict = {"description": [audit_log[5],audit_log[18],audit_log[19]]}
+                audit_details_json = json.dumps(audit_details_dict)
```

Comment:
Would this mean the value of `audit_details_json` is:
```json
{
    "description": "something",
    "category": "something else",
    "activity": "another thing",
    "functional_area": "last thing"
}
```

Reply:
Yep good point, this field as it currently is , is just a dump in 'description' key - but I'm refactoring this to include the remaining fields in the audit table as discussed, so I'll be raising a separate PR for this one :D